R/frequency_encoder.R

```diff
@@ -21,8 +23,19 @@ encode_freq <- function(X) {
 #' my_test,
 #' cat_columns = c("foo"))
 frequency_encoder <- function(X_train, X_test, cat_columns) {
-  train_processed <- dplyr::mutate_if(X_train,~is.element(deparse(substitute(.x)), cat_columns), encode_freq)
```

Comment:
I realized I commented in a previous edit of the function. Anyway, do we need a separate function?. if so, shouldn't we place it after the docstring or as a subfunction?